proposals/infra/INF-0004-inproc-outofproc-compiler-library-support.md

```diff
@@ -0,0 +1,138 @@
+<!-- {% raw %} -->
+
+# In-process only or In-process and Out-of-process support for the compiler library
```

Comment:
"library" is quite overloaded in this space.  Wonder if "API" would more directly indicate what this is trying to solve.

Reply:
Fixed up terminology usage.